src/scripts/create_estner.py

```diff
@@ -0,0 +1,101 @@
+# /// script
+# requires-python = ">=3.10,<4.0"
+# dependencies = [
+#     "datasets==4.0.0",
+#     "huggingface-hub==0.34.4",
+#     "requests==2.32.5",
+# ]
+# ///
+
+"""Create the Estonian NER dataset and upload to HF Hub."""
+from typing import MutableMapping
```

Comment:
Ruff (this should have happened during your pre-commit hook, not sure why it hasn't):

```suggestion

from typing import MutableMapping
```